esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -10,6 +11,62 @@
 from esmf_regrid.experimental.unstructured_regrid import MeshInfo
 
 
+def _map_complete_blocks(src, func, dims, out_sizes):
+    """
+    Apply a function to complete blocks.
+
+    Complete means that the data is not chunked along the chosen dimensions.
```

Comment:
Unsure what this "Complete means", means. Assuming you are referring to the sentence above, might be better to be more explicit here.

Reply:
Bear in mind that this line is taken directly from iris https://github.com/SciTools/iris/blob/226bad38f91d8dddc451411900e90e7777965c75/lib/iris/_lazy_data.py#L358